stix_shifter_modules/alertflex/stix_transmission/api_client.py

```diff
@@ -0,0 +1,29 @@
+import base64
+from stix_shifter_utils.stix_transmission.utils.RestApiClient import RestApiClient
+
+
+class APIClient():
+    PING_TIMEOUT_IN_SECONDS = 10
+
+    def __init__(self, connection, configuration):
+        self.endpoint_start = 'alertflex-ctrl/rest/stix-alerts'
+        headers = dict()
+        auth = configuration.get('auth')
+        headers['Authorization'] = b"Basic " + base64.b64encode(
+            (auth['username'] + ':' + auth['password']).encode('ascii'))
+        self.client = RestApiClient(connection.get('host'),
+                                    connection.get('port'),
+                                    connection.get('cert', None),
+                                    headers,
+                                    cert_verify=False)
```

Comment:
`cert_verify=connection.get('selfSignedCert', False)`

Reply:
Hi,

I have tested alertflex adapter for the master-branch and develop-branch of stix-shifter, the adapter works with master branch but doesn't work with develope-brahch. As far I understand develope-branch has new serious modifications for adapters compare to the master (including selfSignedCert). So maybe I need to move this push request to master branch. What do you recommend?

Reply:
@olegzhr yes we have done some significant changes in develop-branch. we need to incorporate those develop branch changes in new connectors. can you try checking if your branch is up to date with current develop branch?

Reply:
It was downloaded 1 week ago from github

For develope branch stix_shifter shows next error message:

stix_shifter.scripts.stix_shifter ERROR    Module alertflex not found

PyCharm debuger stoped on this string (stix_shifter.py):
connector_module = importlib.import_module("stix_shifter_modules." + module + ".entry_point")

But module exist under this path

Reply:
@olegzhr that's very odd. I pulled down your branch and just ran a simple translate CLI command and it works fine. would you please update your branch to the latest develop again and may be re-create your python virtual environment.